### PR TITLE
Added the pokemon for the Unova and Kalos region

### PR DESCRIPTION
**Pull request by Squirrels**
Hey there!
I'm creating this pull request to add the pokemon from the Unova and Kalos region. To make sure they're up to standards, I followed the following steps:

- Got the pictures from [Teej's Gallery](https://pldh.net/gallery/index) (no Sun and Moon yet, though)
- Compressed them down using ImageOptim's Insane Optimization level
- Got the name and types for each pokemon from [pokeapi.co](https://pokeapi.co/), to prevent manual errors
- Calculated the dark/light threshold using ImageMagick, using the same formula discussed in pull request #66 
- Added them to pokemon.txt, padding the data to follow the same structure
- Added the regions to the specific, random and slideshow options
- Updated regions and max_id's in the database and loaders
- Updated tests and passed all of them
- Added all the pokemon with forms and sizes to Extra (leaving the "default" form in the main folder). 
  - In the case of Deerling/Sawsbuck, I left **spring** as the default form (but also added the spring form to extra, so that _pokemon sawsbuck-spring_ still works). 
  - In the case of Pumpkaboo/Gourgeist, left **average** as the default form (the sizes are **tiny**, **average**, **large** and **super**, since the original images followed that order)
- Updated the README.md to the new max pokemon amount

So yeah! All the ones that actually had a picture were added, leaving the grand total to **719** pokemon.

### CODE REVIEW

File: test_broken.py [hunk 1]
```diff
@@ -9,27 +9,23 @@
 def test_extra_length(region_name='extra'):
     assert len(Database().get_extra()) == expected_len(region_name)
 
-
```
Comment (cclauss) [inline test_broken.py]: Flake8 does not love the blank line counts https://travis-ci.org/LazoCoder/Pokemon-Terminal/builds/265651400#L453 but who cares...  we now have tons more Pokemon!

File: test_database.py [hunk 2]
```diff
@@ -96,6 +101,11 @@ def test_hoenn():
 def test_sinnoh():
     region_test('sinnoh')
 
+def test_sinnoh():
```
Comment (cclauss) [inline test_database.py]: https://travis-ci.org/LazoCoder/Pokemon-Terminal/builds/265651400#L470 This should be `test_unova()` ?  If so then this is worth fixing.

Comment (Squirrels) [inline test_database.py]: Yep! Just fixed it!

File: test_broken.py (hunk 1 above)
Comment (Squirrels) [inline test_broken.py]: Made the appropriate changes to keep flake8 happy.

**Comment by LazoCoder** [comment]
This is excellent. I especially like the fact that you put in the effort to get all the details right, like compressing the images, adding the light/dark thresholds, and all the other little things too. Awesome work, I'll give you credit in the readme.

**Pull request merged**